app/helpers/notification_count.rb

```diff
@@ -5,9 +5,9 @@ def notification_count
         sql = "SELECT COUNT(note.id) AS tally FROM notifications note INNER JOIN submissions sub ON note.iteration_id=sub.id WHERE note.user_id=#{current_user.id} AND note.read='f'"
 
         notifications_count = ActiveRecord::Base.connection.execute(sql).to_a.first["tally"].to_i
-        unconfirmed_memberships_count = current_user.unconfirmed_team_memberships.count
+        memberships_invites_count = current_user.team_membership_invites.count
```

Comment:
I see! ^^ I'm still learning english, so I can relate to that. haha Feel free to point these things out, I really appreciate the feedback. :)